### PR TITLE
Update the progress bar in FullDetailsActivity to the new design

### PR DESCRIPTION
**Pull request by MrChip53**
**Changes**
Updates the progress bar in the FullDetailsActivity to the new design

**Issues**
I had to play around with padding to push the imageview back down to be close to where it originally was so it may not have exact 1:1 placement compared to before.

### CODE REVIEW

**Comment by MrChip53** [comment]
before
after

Before and after images

File: app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/MyDetailsOverviewRow.java [hunk 1]
```diff
@@ -42,6 +43,14 @@ public void setSummary(String mSummary) {
         this.mSummary = mSummary;
     }
 
+    public void setProgress(int progress) {
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/MyDetailsOverviewRow.java:46]:
```suggestion
    public void setProgress(@IntRange(from = 0, to = 100) int progress) {
```

`import androidx.annotation.IntRange;`

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
For series/movies/others with a logo it doesn't work properly - it should not show a progress bar on top of a logo.

**Comment by MrChip53** [comment]
> For series/movies/others with a logo it doesn't work properly - it should not show a progress bar on top of a logo.

I'm not entirely sure what you mean here but I do see it is showing it for series episodes that don't actually have a main image. If this issue goes beyond that, is it reproducible on the demo server?

**Comment by nielsvanvelzen** [comment]
The image shown on the details page is not always the poster/cover. If the item has a logo it will use that. You can test it by addign a logo to a series/movie in the webui and then opening it in the app.

**Comment by MrChip53** [comment]
Should be fixed now. Bad translation so Lint failed though.

File: app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.java [hunk 2]
```diff
@@ -105,6 +108,11 @@ protected void onBindRowViewHolder(RowPresenter.ViewHolder holder, Object item)
 
         vh.mPoster.setImageDrawable(row.getImageDrawable());
         //vh.mStudioImage.setImageDrawable(row.getStudioDrawable());
+        int prog = row.getProgress();
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.java:111]:
```suggestion
        int progress = row.getProgress();
```

File: app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.java [hunk 3]
```diff
@@ -105,6 +108,11 @@ protected void onBindRowViewHolder(RowPresenter.ViewHolder holder, Object item)
...
+        if (row.getProgress() > 0  && vh.mPoster.getDrawable() != null) {
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.java:112]:
```suggestion
        if (prog/*ress*/ > 0  && vh.mPoster.getDrawable() != null) {
```

File: app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.java [hunk 4]
```diff
@@ -105,6 +108,11 @@ protected void onBindRowViewHolder(RowPresenter.ViewHolder holder, Object item)
...
+            vh.mProgress.setProgress(row.getProgress());
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/androidtv/ui/presentation/MyDetailsOverviewRowPresenter.java:113]:
```suggestion
            vh.mProgress.setProgress(prog/*ress*/);
```

File: app/src/main/res/layout/new_details_overview_row.xml [hunk 5]
```diff
@@ -150,14 +150,36 @@
...
-            android:layout_height="match_parent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             app:srcCompat="@drawable/blank30x30"
             android:layout_marginTop="60dp"
             android:foregroundGravity="fill_horizontal"
-            android:scaleType="centerInside"
-            android:padding="50dp"
-            android:layout_centerHorizontal="true"
+            android:scaleType="centerCrop"
```
Comment (nielsvanvelzen) [inline app/src/main/res/layout/new_details_overview_row.xml:158]: Should keep centerInside because otherwise logo's would get cut-off

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

**Review by nielsvanvelzen — APPROVED** [review]
Code looks fine, need to test it before merging.

**Pull request merged**